pkg/chartrepotest/chartmuseumreal.go

```diff
@@ -45,7 +45,7 @@ func tChartMuseumReal(t *testing.T, username, password string) (string, func())
 		id,
 	))
 
-	return "http://localhost:" + port, func() {
+	return "http://127.0.0.1:" + port, func() {
```

Comment:
Out of curiosity, why did you need this?

Reply:
I had some tests failing and I thought this was related. At the end, it was another thing but I kept the change.